sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/DataSourceReadBenchmark.scala

```diff
@@ -541,6 +548,9 @@ object DataSourceReadBenchmark extends SqlBasedBenchmark {
   }
 
   override def runBenchmarkSuite(mainArgs: Array[String]): Unit = {
+    runBenchmark("SQL Single Boolean Column Scan") {
+      numericScanBenchmark(1024 * 1024 * 15, BooleanType)
```

Comment:
I wonder if we can just add `BooleanType` to the benchmark below, together with other types.

Reply:
Updated